### PR TITLE
セッションIDが保存されていない場合カートにアクセスできない問題を解決

### PR DESCRIPTION
**Pull request by tkeshun**

### CODE REVIEW

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 1]
```diff
@@ -48,6 +48,8 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
 		}
 
 		Integer accessId = null;
+		Integer sessionId = null;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:51]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 2]
```diff
@@ -48,6 +48,8 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+		// userの有無
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 3]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
 				accessId = Integer.parseInt(getId.toString());
 			}
 		} else {
+			// sessionIdで保存されたやつをUpdateする
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 4]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
 			accessId = user.getId();
-			String sessionId = httpSession.getAttribute("accessId").toString();
-			System.out.println("UPDATE::::");
-			cartService.updateUserId(accessId, Integer.parseInt(sessionId));
+			Object checkId = httpSession.getAttribute("accessId");// 保存してあるsessionId
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 24, expected level should be 6.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>';' is not followed by whitespace.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 5]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			if (checkId == null) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 6]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				sessionId = httpSession.getId().hashCode();
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 7]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				httpSession.setAttribute("accessId", sessionId);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 8]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			} else {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 9]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				sessionId = Integer.parseInt(checkId.toString());
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:69]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 10]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+				cartService.updateUserId(accessId, sessionId);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:70]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else' child has incorrect indentation level 32, expected level should be 8.

File: src/main/java/com/example/controller/ShoppingCartController.java [hunk 11]
```diff
@@ -57,10 +59,16 @@ public String showCart(Model model, @AuthenticationPrincipal LoginUser loginUser
...
+			}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/ShoppingCartController.java:71]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'else rcurly' has incorrect indentation level 24, expected level should be 6.

**Review by uedamiku — APPROVED** [review]
問題ないと思います。

**Pull request merged**